src/main/java/org/folio/inventory/resources/Instances.java

```diff
@@ -1,5 +1,16 @@
 package org.folio.inventory.resources;
 
+import io.vertx.core.CompositeFuture;
```

Comment:
We typically follow the Eclipse style ordering of imports, see https://issues.folio.org/browse/FOLIO-1667?focusedCommentId=41677&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-41677 for more details

Please put these import statements back to the end of the list

Reply:
@marcjohnson-kint fixed imports